pages/api/popular/api_popular.js

```diff
@@ -0,0 +1,8 @@
+export default async function handler(request, response) {
+  // const { page } = request.query;
+  const apiResponse = await fetch(
+    `https://api.themoviedb.org/3/movie/popular?api_key=${process.env.API_KEY}&language=en-US&page=1`
+  );
+  const movieData = await apiResponse.json();
+  response.status(200).json(movieData.results);
+}
```

Comment:
All of these api routes use the same handler fucntion, this is a GREAT case to outsource this as a fetcher function in a service folder.
For that you can create a new folder called services and in there is a folder called fetch. In the index.js of that folder you can write a fetcher function that takes the URL (which are different in your handler functions) of your functions as arguments.
Remember: We want to never repeat ourselves in our code :)